### PR TITLE
commands/service:gen-doc: add marketplace related instructions

### LINKED ISSUE #951: Generated README should introduce the marketplace
The generated README should introduce the marketplace and display link to it for the deploy instead of the repo.
It could be based on the service's sid.

EG: https://github.com/mesg-foundation/service-ethereum-erc20#purchase-the-service

### PR DESCRIPTION
**Pull request by ilgooz**
resolves #951

### CODE REVIEW

File: commands/provider/assets/readme_template.md [hunk 1]
```diff
@@ -23,9 +23,24 @@ You can install MESG Core by running the following command or [follow the instal
 bash <(curl -fsSL https://mesg.com/install)
 ```
 
-## Service
+{{if .Sid}}
+## Purchase the service
```
Comment (antho1404) [inline commands/provider/assets/readme_template.md]: This is an optional step, I would keep it simple with just the link of the website with something like.

To know how to deploy this service please follow the steps on this link https://marketplace.mesg.com/services/sid.

Can find a better text but something like that.

File: commands/provider/assets/readme_template.md [hunk 2]
```diff
@@ -23,9 +23,24 @@ You can install MESG Core by running the following command or [follow the instal
...
 
-{{if .Repository}}To deploy this service, run the following command:
+To deploy this service, you first need to purchase one of its offers from the MESG Marketplace.
```
Comment (NicolasMahe) [inline commands/provider/assets/readme_template.md]:
```suggestion
To deploy this service, you first need to purchase one of its offers from the [MESG Marketplace](https://marketplace.mesg.com/).
```

File: commands/provider/assets/readme_template.md [hunk 3]
```diff
@@ -23,9 +23,24 @@ You can install MESG Core by running the following command or [follow the instal
...
+
+To do so, go to the service's page on the MESG Marketplace and check the offers tab: https://marketplace.mesg.com/services/{{.Sid}}#offers
+{{end}}
+
+## Deploy the service
```
Comment (NicolasMahe) [inline commands/provider/assets/readme_template.md]: I agree with https://github.com/mesg-foundation/core/pull/954/files#r283428257, can you remove the `## Deploy the service` block to only keep `## Purchase the service`.

I think it's best to simply remove the deploy for git url to only show the marketplace deploy

Comment (NicolasMahe) [reply commands/provider/assets/readme_template.md]: Just like: https://github.com/mesg-foundation/service-ethereum-erc20#purchase-the-service

**Comment by NicolasMahe** [comment]
Closing this PR in favor of https://github.com/mesg-foundation/core/pull/960

**Pull request closed without merging**